compiler/src/iree/compiler/Codegen/Common/GPU/GPUHeuristics.h

```diff
@@ -38,14 +41,33 @@ struct GPUMMAHeuristicSeeds {
 struct GPUMMASchedule {
   // Index of the chosen intrinsic into the list of given MMA intrinsics
   uint64_t index;
-  int64_t mSize;      // Native MMA size along M dimension
-  int64_t nSize;      // Native MMA size along N dimension
-  int64_t kSize;      // Native MMA size along K dimension
-  int64_t mWarpCount; // Number of subgroups along M dimension
-  int64_t nWarpCount; // Number of subgroups along N dimension
-  int64_t mTileCount; // Number of tiles per subgroup along M dimension
-  int64_t nTileCount; // Number of tiles per subgroup along N dimension
-  int64_t kTileCount; // Number of tiles along K dimension
+  int64_t mSize; // Native MMA size along M dimension
+  int64_t nSize; // Native MMA size along N dimension
+  int64_t kSize; // Native MMA size along K dimension
```

Comment:
What does "native" mean now? Before, it made sense since it was the only dimension.

Reply:
It is still the same, since there is only 1 mSize, nSize, and kSize (corresponding to the target intrinsic subgroup sizes). I will clarify in the comments.